### ISSUE TITLE
Does the pi connects usb?

### ISSUE THREAD

**Issue by kaisarantonn** [toannv28/Klipper-Prusa#1]
Hello! Sorry im a noob on this pi thing.

How to flash the firmware of einsy? Does it same like ender 3 by sdcard?

And how does the pi connects with einsy? By USB Pi to usb on Einsy?

Thank you!

**Comment by toannv28** [comment]
Flash Klipper to your printer https://www.klipper3d.org/Installation.html#building-and-flashing-the-micro-controller

**Comment by toannv28** [comment]
> Hello! Sorry im a noob on this pi thing.
> 
> How to flash the firmware of einsy? Does it same like ender 3 by sdcard?
> 
> And how does the pi connects with einsy? By USB Pi to usb on Einsy?
> 
> Thank you!

Flash Klipper to your printer https://www.klipper3d.org/Installation.html#building-and-flashing-the-micro-controller

**Comment by toannv28** [comment]
> Hello! Sorry im a noob on this pi thing.
> 
> How to flash the firmware of einsy? Does it same like ender 3 by sdcard?
> 
> And how does the pi connects with einsy? By USB Pi to usb on Einsy?
> 
> Thank you!

To use this config, the firmware should be compiled for the AVR atmega2560. To use via serial, in "make menuconfig" select "Enable extra low-level configuration options" and select serial1 (the RasPi serial) or serial0 when you plan to connect via the USB.
And Flash Klipper to your printer https://www.klipper3d.org/Installation.html#building-and-flashing-the-micro-controller

**Comment by kaisarantonn** [comment]
> > Hello! Sorry im a noob on this pi thing.
> 
> > 
> 
> > How to flash the firmware of einsy? Does it same like ender 3 by sdcard?
> 
> > 
> 
> > And how does the pi connects with einsy? By USB Pi to usb on Einsy?
> 
> > 
> 
> > Thank you!
> 
> 
> 
> To use this config, the firmware should be compiled for the AVR atmega2560. To use via serial, in "make menuconfig" select "Enable extra low-level configuration options" and select serial1 (the RasPi serial) or serial0 when you plan to connect via the USB.
> 
> And Flash Klipper to your printer https://www.klipper3d.org/Installation.html#building-and-flashing-the-micro-controller

How to flash firmware back to stock? Is it just connect via usb and use prusaslicer as usual?

**Comment by toannv28** [comment]
> > > Hello! Sorry im a noob on this pi thing.
> > 
> > 
> > > 
> > 
> > 
> > > How to flash the firmware of einsy? Does it same like ender 3 by sdcard?
> > 
> > 
> > > 
> > 
> > 
> > > And how does the pi connects with einsy? By USB Pi to usb on Einsy?
> > 
> > 
> > > 
> > 
> > 
> > > Thank you!
> > 
> > 
> > To use this config, the firmware should be compiled for the AVR atmega2560. To use via serial, in "make menuconfig" select "Enable extra low-level configuration options" and select serial1 (the RasPi serial) or serial0 when you plan to connect via the USB.
> > And Flash Klipper to your printer https://www.klipper3d.org/Installation.html#building-and-flashing-the-micro-controller
> 
> How to flash firmware back to stock? Is it just connect via usb and use prusaslicer as usual?

Yes, just need use prusaslicer and flash firmware prusa, the board will to back stock :)
